docs/modules/user-guide/partials/proc_building-a-devfile-registry.adoc

```diff
@@ -0,0 +1,47 @@
+[id="building-a-custom-devfile-registry_{context}"]
+= Building a custom devfile registry
+
+[role="_abstract"]
+Make your repository into a devfile registry by building a `devfile index` container image. Deploy the `devfile index` image to your cloud or cluster to form the devfile registry.
```

Comment:
`devfile index` is formatted here, but is not in the `proc_tagging-and-pushing-a-registry-container-image.adoc` file.

I wonder here where the focus is supposed to be. Is it a "devfile index" image, or is it a devfile "index image"? I'm not sure if that makes any sense, but I'm wondering how we can use that information to decide on whether or not we need to format this particular phrase.